src/components/course/data/hooks.js

```diff
@@ -237,6 +237,21 @@ useCoursePriceForUserSubsidy.propTypes = {
   }).isRequired,
 };
 
+/**
```

Comment:
Hooray for docstrings!!!

Reply:
+1 Appreciate the efforts to add docstrings to our MFEs 😄